bindings/twilio/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,129 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package sendgrid
```

Comment:
Please open a doc issue for this as well.

Reply:
Just checking do you want me to set up a PR with some extra docs, or just an issue to discuss it?

Reply:
Doc issue to start with. PR would be great!

Reply:
@amanbha Hi, I've created a PR for the docs. Let me know if you need an issue to track it too https://github.com/dapr/docs/pull/535

Reply:
Great @benc-uk Thanks. I am merging it, please make the corresponding changes in dapr runtime as well.